util/raclgen/lib.py

```diff
@@ -50,3 +57,52 @@ def compute_policy_value(permission: str) -> int:
             policy['rd_default'] = compute_policy_value('allowed_rd')
             policy['wr_default'] = compute_policy_value('allowed_wr')
     return racl_config
+
+
+def parse_racl_mapping(racl_config: Dict[str, object], mapping_path: str,
+                       if_name: Optional[str], ip_block: IpBlock):
+    mapping = _read_hjson(mapping_path)
+    parsed_register_mapping = {}
+
+    # Mapping must be a dict with a single entry:
+    # RACL_GROUP => register mapping
+    if len(mapping) != 1:
+        raise SystemExit('Mapping file must be a single-element dict mapping the RACL group to the '
```

Comment:
Apparently you have not run util/lintpy.py on this file: it will suggest reformatting this line, and a few others perhaps. It is good practice to run it since it makes the code compliant with the style guide.